docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -1089,6 +1089,18 @@ _Returns_
 
 -   `boolean`: Is navigation mode enabled.
 
+### isRemovalPromptDisplayed
+
+Selector that returns if the removal prompt needs to be displayed.
```

Comment:
The selector returns an object, so its result is always truthy, which could be confusing.

Reply:
Agreed, we should change/rename this.